cli/central-client/src/main/java/org/ballerinalang/central/client/Utils.java

```diff
@@ -274,6 +278,17 @@ private static void writeAndHandleProgress(InputStream inputStream, FileOutputSt
         }
     }
 
+    private static void writeAndHandleProgressWithoutLogging(InputStream inputStream, FileOutputStream outputStream)
+            throws IOException {
+        int count;
+        byte[] buffer = new byte[1024];
+
+        while ((count = inputStream.read(buffer)) > 0) {
+            outputStream.write(buffer, 0, count);
```

Comment:
We should not write to the output stream. The idea is that we do the pull operation quietly. If the pull fails due to any interruption, then we should throw an exception and log it as a diagnostic.